blog/2017-08-10-RealTimePlanningUsingKieServerPart1.adoc

```diff
@@ -0,0 +1,149 @@
+= Optimize your problems using KIE Server, part 1
+:page-interpolate: true
+:awestruct-author: mcimbora
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, howto, kie server]
```

Comment:
Can we change the "kie server" tag to "execution server", which is more inline with the website tabs and documentation sections? We probably want a "workbench" tag too.